src/config/config.ts

```diff
@@ -14,6 +15,7 @@ import { sfdc } from '../util/sfdc';
 import { ConfigFile } from './configFile';
 import { ConfigContents, ConfigValue } from './configStore';
 
+const debug = Debug('@salesforce/core:config');
```

Comment:
We typically use Logger for debugging in this library (which will also delegate to the debug library). i.e. `Logger.childFromRoot('core:config')`